homeassistant/components/vesync/quality_scale.yaml

```diff
@@ -0,0 +1,66 @@
+rules:
+  # Bronze
+  action-setup: todo
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage:
+  config-flow: done
+  dependency-transparency: done
+  docs-actions: done
+  docs-high-level-description: todo
+  docs-installation-instructions: todo
+  docs-removal-instructions: todo
+  entity-event-setup:
+    status: exempt
+    comment: No events used.
+  entity-unique-id: done
+  has-entity-name: done
+  runtime-data: todo
+  test-before-configure: done
+  test-before-setup: todo
+  unique-config-entry: done
+
+  # Silver
+  action-exceptions: todo
+  config-entry-unloading: done
+  docs-configuration-parameters:
+    status: exempt
+    comment: No configuration parameters.
+  docs-installation-parameters:
+    status: exempt
+    comment: No installation parameters beyond username password.
+  entity-unavailable: done
```

Comment:
We should also incorporate `super().available` to go unavailable when the coordinator update fails